OsmAnd-java/src/main/java/net/osmand/router/RoutingConfiguration.java

```diff
@@ -469,7 +469,17 @@ private static boolean checkTag(String pname) {
 	private static void addSubclause(RoutingRule rr, RouteAttributeContext ctx) {
 		boolean not = "ifnot".equals(rr.tagName);
 		if(!Algorithms.isEmpty(rr.param)) {
-			ctx.getLastRule().registerAndParamCondition(rr.param, not);
+			if (rr.param.contains(",")) {
+				String[] params = rr.param.split(",");
+				for (String p : params) {
+					p = p.trim();
+					if (!Algorithms.isEmpty(p)) {
+						ctx.getLastRule().registerAndParamCondition(p, not);
+					}
+				}
+			} else {
+				ctx.getLastRule().registerAndParamCondition(rr.param, not);
+			}
```

Comment:
It could be simplified with just-split-iterator, without contains() and if() block.

Additionally, it's need to verify that `if=motorcycle, avoid_motorway, prefer_unpaved` would work as:

if (motorcycle **AND** avoid_motorway **AND** prefer_unpaved)

Reply:
if (motorcycle AND avoid_motorway AND prefer_unpaved) -- verifed

It's OK to accept this request.